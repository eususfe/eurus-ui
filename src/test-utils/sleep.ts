/*
 * @Author: jiangxd
 * @Date: 2022-09-23 14:06:55
 * @LastEditTime: 2022-09-23 14:07:04
 * @LastEditors: jiangxd
 * @Description:
 * @FilePath: /eurus-ui/src/test-utils/sleep.ts
 */
const sleep = (time = 0) => new Promise(resolve => setTimeout(resolve, time));

export default sleep;
