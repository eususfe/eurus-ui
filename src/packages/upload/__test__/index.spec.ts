import { mount } from '@vue/test-utils';
import EUpload from '..';

describe('EUpload', () => {

  it('EUpload snapshot', () => {
    const wrapper = mount(EUpload);
    expect(wrapper.html()).toMatchSnapshot();
    wrapper.unmount();
  });
});
