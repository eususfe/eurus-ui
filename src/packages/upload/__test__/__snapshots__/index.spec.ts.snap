// Vitest Snapshot v1

exports[`EUpload > EUpload snapshot 1`] = `
"<div class=\\"e-upload\\">
  <div class=\\"e-upload-content\\">
    <!----><input type=\\"file\\" class=\\"file\\" title=\\"\\">
  </div>
  <div class=\\"e-upload-list\\"></div>
</div>"
`;
